setup.py

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/python
+import os
+from shinken.bin import VERSION
```

Comment:
Hum, in this case this I think python is doing a relative import . Setup.py is next to the shinken directory in the repo